pkg/api/helpers.go

```diff
@@ -58,16 +58,10 @@ var Semantic = conversion.EqualitiesOrDie(
 		// TODO: if we decide it's important, it should be safe to start comparing the format.
 		//
 		// Uninitialized quantities are equivalent to 0 quantities.
-		if a.Amount == nil && b.MilliValue() == 0 {
+		if a.Sign() == 0 && b.Sign() == 0 {
```

Comment:
It is, except that I didn't know whether I wanted to treat nil
quantity as zero by default in compare.  Will change.